### ISSUE TITLE
finetuning question

### ISSUE THREAD

**Issue by solemnrole** [sanghoon/pva-faster-rcnn#57]
hi,@sanghoon,there is a question when i trained your net,i using the 'pvanet_obsolete'version ,when i funetuning the net using the VOC07 dataset, i found the loss is very big,but when i dont implement the boxes regression,it turned normal,i dont understand, would you mind give me some explaination?thanks for your reading.

**Comment by yzhang123** [comment]
maybe the OBSOLETE version was created without the intention of a bbox regression
